client/src/components/AuthHeader/AuthHeader.tsx

```diff
@@ -2,22 +2,19 @@ import Button from '@material-ui/core/Button';
 import Box from '@material-ui/core/Box';
 import { Link } from 'react-router-dom';
 import useStyles from './useStyles';
-import { Typography } from '@material-ui/core';
 
 interface Props {
   linkTo: string;
-  asideText: string;
   btnText: string;
 }
 
-const AuthHeader = ({ linkTo, asideText, btnText }: Props): JSX.Element => {
+const AuthHeader = ({ linkTo, btnText }: Props): JSX.Element => {
   const classes = useStyles();
 
   return (
     <Box p={1} className={classes.authHeader}>
-      <Typography className={classes.accAside}>{asideText}</Typography>
       <Link to={linkTo} className={classes.link}>
-        <Button color="inherit" className={classes.accBtn} variant="contained">
+        <Button className={classes.accBtn} variant="outlined">
           {btnText}
         </Button>
```

Comment:
It would have been nice to have these styled to match the mocks closer - the buttons having the right variants and colors, [Material-UI palette colors](https://material-ui.com/customization/palette/) adjusted accordingly. Some of the styling might not be working as intended since this `<Button>` is wrapped in a `react-router-dom` link, see the comment above for how to resolve.

Reply:
Rajiv linked me with a resource to fix this!